app/helpers/react_components/mentoring/mentor_discussion.rb

```diff
@@ -0,0 +1,47 @@
+module ReactComponents
+  module Mentoring
+    class MentorDiscussion < ReactComponent
+      initialize_with :discussion
+
+      def to_s
+        scratchpad = ScratchpadPage.new(about: discussion.solution.exercise)
+        student = discussion.solution.user
+        track = discussion.solution.track
+        exercise = discussion.solution.exercise
+
+        super(
+          "mentoring-mentor-discussion",
+          {
+            discussion_id: discussion.uuid,
+            student: {
+              avatar_url: student.avatar_url,
+              handle: student.handle
+            },
+            track: {
+              title: track.title,
+              icon_url: track.icon_url
+            },
+            exercise: {
+              title: exercise.title
+            },
+            iterations: discussion.solution.iterations.map do |iteration|
+              {
+                idx: iteration.idx,
+                num_comments: Solution::MentorDiscussionPost.where(discussion: discussion, iteration: iteration).count,
+                # TODO: @iHiD, I don't know how to get the correct unread state
```

Comment:
@iHiD pinging you here just in case you don't see this.

Reply:
Me neither :) 

Thanks. I'll work it out tomorrow.